docs/guide/controllers.md

```diff
@@ -753,7 +753,11 @@ When pagination is enabled, `items` is an instance of PagerFanta:
 
 ```
 {% if isPaginationEnabled %}
-    {{ pagerfanta( items, 'ez', {'routeName': location, 'pageParameter': pageParameter } ) }}
+    {{ pagerfanta( items, 'ez', {
+        'pageParameter': pageParameter,
+        'routeName': '_ez_content_view',
+        'routeParams' : {'contentId': content.id, 'locationId': location.id }
+    } ) }}
```

Comment:
Actually, it should be documented that way:

```suggestion
    {{ pagerfanta( items, 'ez', {
        'pageParameter': pageParameter,
        'routeName': 'ez_urlalias',
        'routeParams' : {'location': location }
    } ) }}
```

A follow-up will change the template used by the frontend that way (we need `_ez_content_view` in the backoffice, but for now it is used for both back & frontend).